docs/main/user-guides/replicated-storage-user-guide/rs-deployment.md

```diff
@@ -0,0 +1,244 @@
+---
+id: rs-deployment
+title: Deploy an Application
+keywords: 
+  - Deployment
+  - Deploy an Application
+description: This guide will help you to deploy OpenEBS Replicated Storage.
+---
+
+# Deploy an Application
+
+If all verification steps in the preceding stages were satisfied, then Replicated Storage (a.k.a Replicated Engine and f.k.a Mayastor) has been successfully deployed within the cluster. In order to verify basic functionality, we will now dynamically provision a Persistent Volume based on a Replicated StorageClass, mount that volume within a small test pod which we'll create, and use the [**Flexible I/O Tester**](https://github.com/axboe/fio) utility within that pod to check that I/O to the volume is processed correctly.
+
+## Define the PVC
+
+Use `kubectl` to create a PVC based on a StorageClass that you created in the [previous stage](../replicated-storage-user-guide/rs-configuration.md#create-replicated-storageclasss). In the example shown below, we will consider that StorageClass to have been named "mayastor-1". Replace the value of the field "storageClassName" with the name of your own Replicated Storage-based StorageClass.
+
+For the purposes of this quickstart guide, it is suggested to name the PVC "ms-volume-claim", as this is what will be illustrated in the example steps which follow.
+
+**Command**
+
+```
+cat <<EOF | kubectl create -f -
+apiVersion: v1
+kind: PersistentVolumeClaim
+metadata:
+  name: ms-volume-claim
+spec:
+  accessModes:
+  - ReadWriteOnce
+  resources:
+    requests:
+      storage: 1Gi
+  storageClassName: mayastor-1
+EOF
+```
+
+**YAML**
+
+```
+apiVersion: v1
+kind: PersistentVolumeClaim
+metadata:
+  name: ms-volume-claim
+spec:
+  accessModes:
+  - ReadWriteOnce
+  resources:
+    requests:
+      storage: 1Gi
+  storageClassName: INSERT_YOUR_STORAGECLASS_NAME_HERE
+```
+
+If you used the storage class example from previous stage, then volume binding mode is set to `WaitForFirstConsumer`. That means, that the volume won't be created until there is an application using the volume. We will go ahead and create the application pod and then check all resources that should have been created as part of that in kubernetes.
+
+## Deploy the FIO Test Pod
+
+The Replicated Storage CSI driver will cause the application pod and the corresponding Replicated Storage volume's NVMe target/controller ("Nexus") to be scheduled on the _same_ Replicated Storage Node, in order to assist with restoration of volume and application availabilty in the event of a storage node failure.
+
+:::warning
```

Comment:
This limitation no longer exists IMO. @Abhinandan-Purkait can you please confirm?

Reply:
That's right, we no longer use the local parameter, atleast at the moment. Might come into picture if we want to introduce the local offering in mayastor. 

Let's remove this warning section for now?

Reply:
Can we remove the warning? @avishnu @Abhinandan-Purkait